cloudbuild.yaml

```diff
@@ -150,22 +150,23 @@ steps:
         -var=project_id=$PROJECT_ID \
         -var=cluster_name=ml-$SHORT_SHA-$_PR_NUMBER-$_BUILD_ID-cluster \
         -var=cluster_location=$_REGION \
-        -var=kubernetes_namespace=ml-$SHORT_SHA-$_BUILD_ID \
+        -var=kubernetes_namespace=ml-$SHORT_SHA-$_BUILD_ID-jupyter \
         -var=workload_identity_service_account=jupyter-sa-$SHORT_SHA-$_BUILD_ID \
         -var=gcs_bucket=gke-aieco-jupyter-$SHORT_SHA-$_BUILD_ID \
         -auto-approve -no-color -lock=false
         echo "pass" > /workspace/jupyterhub_tf_result.txt
 
-        kubectl wait --all pods -n ml-$SHORT_SHA-$_BUILD_ID --for=condition=Ready --timeout=1200s
-        kubectl get services -n ml-$SHORT_SHA-$_BUILD_ID
-        kubectl port-forward -n ml-$SHORT_SHA-$_BUILD_ID service/proxy-public 9443:80 &
+        kubectl wait --all pods -n ml-$SHORT_SHA-$_BUILD_ID-jupyter --for=condition=Ready --timeout=1500s
+        kubectl get services -n ml-$SHORT_SHA-$_BUILD_ID-jupyter
+        kubectl port-forward -n ml-$SHORT_SHA-$_BUILD_ID-jupyter service/proxy-public 9442:80 &
         # Wait port-forwarding to take its place
         sleep 5s
 
         cd /workspace/modules/jupyter/tests
-        python3 test_hub.py "127.0.0.1:9443" $_AUTOPILOT_CLUSTER
+        python3 test_hub.py "127.0.0.1:9442" $_AUTOPILOT_CLUSTER
         echo "pass" > /workspace/jupyterhub_test_result.txt
     allowFailure: true
+    waitFor: ['create gke cluster']
```

Comment:
Update 'test jupyterhub' too?

Reply:
This is weird, how was this working before without a `waitFor`?

Reply:
So if there is no explicit `waitFor` it will just wait for all previous steps to finish before running.

Reply:
For more info: https://cloud.google.com/build/docs/configuring-builds/configure-build-step-order#build_step_order_and_dependencies